scripts/multinode/configure-pri.sh

```diff
@@ -43,6 +43,9 @@ function configure_microshift() {
     sleep 5
     sudo systemctl start microshift-ovs-init.service
 
+    # Add dummy IP to br-ex
+    sudo ip addr add 10.44.0.0/32 dev br-ex
```

Comment:
Is this call idempotent? In other words, if we run the script on the same machine twice, does it work for the second time without failing?

Reply:
Also, can we please add a comment in the script explaining why this is done?

Reply:
fixed. PTAL